src/main/scala/com/cognite/sdk/scala/common/ReadableResource.scala

```diff
@@ -9,11 +9,21 @@ import io.circe.{Decoder, Encoder}
 import io.circe.derivation.deriveEncoder
 
 // TODO: Verify that index and numPartitions are valid
+
+object listLimits {
```

Comment:
Should be `listLimits`, however: I think we should call this parameter "batchSize" instead, similar to what we do in https://github.com/cognitedata/cdp-spark-datasource#common-options